src/components/RequestList.jsx

```diff
@@ -1,8 +1,8 @@
 import React, { useState } from 'react';
-import { DropdownButton, Dropdown, Card } from 'react-bootstrap';
+import { DropdownButton, Dropdown, Card, Button, Modal } from 'react-bootstrap';
```

Comment:
this change needs to be reverted. Modal is included in Modal.jsx and called from this file now.